docs/docs/coverage/iac/azure-arm.md

```diff
@@ -0,0 +1,31 @@
+# Azure ARM Template
+Trivy supports the scanners listed in the table below.
+
+|      Scanner       | Supported |
+| :----------------: | :-------: |
+| [Misconfiguration] |     ✓     |
+|      [Secret]      |     ✓     |
+
+It supports the following configurations:
+
+|    Format    | Supported |
+| :----------: | :-------: |
+| ARM template |     ✓     |
+|    Bicep     |     ✓     |
```

Comment:
Can we say that Trivy supports `bicep` configuration if the user needs to convert bicep to ARM himself and as a result Trivy will scan ARM?

Reply:
I'd say partially. Is there any symbol describing that other than `✓`, `x` or `-`?

Reply:
I've added footnotes. Please let me know if you have any ideas.
https://github.com/aquasecurity/trivy/pull/4954/commits/35e5532f0ce633545c91de00d5b636b630d361fb